Neos.Neos/webpack.js.config.js

```diff
@@ -0,0 +1,56 @@
+const debug = process.env.NODE_ENV !== "production";
+const TerserPlugin = require("terser-webpack-plugin");
+const webpack = require("webpack");
+
+const javascriptConfig = {
+	context: __dirname,
+	devtool: debug ? 'source-map' : false,
```

Comment:
I think since webpack 4 you can define devtool but set `mode` to NODE_ENV and webpack will do the right thing.